services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -113,7 +113,7 @@
                 "id": "arp-a5b",
                 "type": "date",
                 "hydrate": "reportingPeriodEndDate",
-                "validation": "dateOptional",
+                "validation": "textOptional",
```

Comment:
Why the swap here?

Reply:
Uh... I can't remember. I'm thinking they should be date. I will change that.